SBTi/__init__.py

```diff
@@ -2,4 +2,74 @@
 This package helps companies and financial institutions to assess the temperature alignment of current
 targets, commitments, and investment and lending portfolios, and to use this information to develop
 targets for official validation by the SBTi.
-"""
\ No newline at end of file
+"""
```

Comment:
hmm, I would try to avoid putting more code than import/export statements into the `__init__.py` files, rather make a separate file/class for that. If you want to have nicer import statements, you can still add the imports to this file